src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/Validation/ValidatorItem.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Mvc.ModelBinding.Validation
+{
+    public class ValidatorItem
+    {
+        public ValidatorItem()
```

Comment:
This is for mocking in some tests and stuff like these https://github.com/aspnet/Mvc/pull/3990/files#diff-4538fda6cd1ba9234665211044c79245R95